### PR TITLE
Follow-up for #3326

### PR DESCRIPTION
**Pull request by glitsj16**
Purely cosmetic stuff, no functional changes.

### CODE REVIEW

File: etc/profile-a-l/chromium-common.profile [hunk 1]
```diff
@@ -40,5 +39,9 @@ disable-mnt
 ?BROWSER_DISABLE_U2F: private-dev
 # private-tmp - problems with multiple browser sessions
 
+# prevents access to passwords saved in GNOME Keyring and KWallet, also breaks Gnome connector
+# dbus-user none
+# dbus-system none
```
Comment (rusty-snake) [inline etc/profile-a-l/chromium-common.profile:44]: Can we block the system-bus?

Comment (glitsj16) [reply etc/profile-a-l/chromium-common.profile:44]: @rusty-snake Not sure, I'm actually not that familiar with chrome-based browsers. So if you say the system bus can be blocked without issues, I'm ok with that. Thanks for the input, I didn't want to commit this without seeking some advice.

Comment (rusty-snake) [reply etc/profile-a-l/chromium-common.profile:44]: Password are per user, therefore on the session-bus. 

In general: I block the system-bus in my `pre-globals.local` for around two month now, and only allowed it for d-feet and searhorse. IHMO we can add `dbus-system none` to the most profiles.

Comment (rusty-snake) [reply etc/profile-a-l/chromium-common.profile:44]: But I'm not sure if it maybe still breaks something in chromium.

Comment (glitsj16) [reply etc/profile-a-l/chromium-common.profile]: @rusty-snake That makes sense, thanks for your input, as always. I've blocked the system bus, we can change this later if we get issue reports.

**Pull request merged**